x.py

```diff
@@ -279,17 +279,17 @@ def package(mode: str, package_path: str):
         (f"target/{mode}/prusti-server*", "."),
         (f"target/{mode}/prusti-rustc*", "."),
         (f"target/{mode}/cargo-prusti*", "."),
-        (f"prusti-contracts/target/verify/{mode}/libprusti_contracts.*", "."),
-        (f"prusti-contracts/target/verify/{mode}/deps/libprusti_contracts_proc_macros-*", "deps"),
-        (f"prusti-contracts/target/verify/{mode}/deps/prusti_contracts_proc_macros-*.dll", "deps"),
-        (f"prusti-contracts/target/verify/{mode}/libprusti_std.*", "."),
-        (f"prusti-contracts/target/verify/{mode}/deps/libprusti_contracts-*", "deps"),
-        (f"prusti-contracts/target/verify/{mode}/deps/prusti_contracts-*.dll", "deps"),
+        (f"target/verify/{mode}/libprusti_contracts.*", "."),
```

Comment:
Uh, actually, why is `verify` in this path?  It might actually be `target/{mode}/`, here and below.

Reply:
`cargo-prusti` always adds a `verify` to any cargo target dir you give it, so the `target/verify/...` with prusti is the equivalent of `target/...` for cargo